Description: Don't build and install smtpscript
 smtpscript is a utility used for regression testing and doesn't need to be
 installed in a package, as confirmed with upstream.
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/mk/regress/Makefile.am
===================================================================
--- opensmtpd.orig/mk/regress/Makefile.am	2013-12-22 09:33:51.459036681 -0500
+++ opensmtpd/mk/regress/Makefile.am	2013-12-23 09:59:31.367653536 -0500
@@ -1,6 +1,6 @@
 include	$(top_srcdir)/mk/pathnames
 
-bin_PROGRAMS=		smtpscript
+bin_PROGRAMS=
 
 smtpscript_SOURCES=	$(regress_srcdir)/smtpscript.c
 smtpscript_SOURCES+=	$(regress_srcdir)/iobuf.c
